src/bin/cargo/commands/test.rs

```diff
@@ -130,8 +129,8 @@ pub fn exec(config: &mut Config, args: &ArgMatches<'_>) -> CliResult {
         compile_opts,
     };
 
-    // TESTNAME is actually an argument of the test binary, but it's
-    // important so we explicitly mention it and reconfigure
+    // `TESTNAME` is actually an argument of the test binary, but it's
+    // important, so we explicitly mention it and reconfigure.
```

Comment:
Do we prefer non-doc comments to end in full stops?

Reply:
Good question. For me, this is two points: a) sentences should always end in full stops, b) it's still useful to end phrases (not proper sentences) in full stops in case the following line is some commented-out code (that way you know exactly where the comment itself ends), and it's also useful when a full sentence follows the phrase. Finally, consistency with the Rustc codebase.